### PR TITLE
[REF] product_customer_code, product_supplierinfo_for_customer: Changes in README

### PR DESCRIPTION
**Pull request by ysantiago**

### CODE REVIEW

**Review by moylop260 — APPROVED** [review]
👍 
This README update help us to avoid create changes that won't be approved.

(Like as us from https://github.com/OCA/product-attribute/pull/276)

**Comment by moylop260** [comment]
@JulioSerna @elicoidal What do you think about?

**Review by JulioSerna — APPROVED** [review]

File: product_customer_code/README.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    :alt: License: AGPL-3
+
+=====================
+Product customer code
+=====================
+
+This module extends of module product and allows to have different codes
+for the same product according to the customer.
+
+**NOTE**: This module is overlapped by `product_supplierinfo_for_customer
```
Comment (elicoidal) [inline product_customer_code/README.rst]: do you mean superseded?

File: product_customer_code/README.rst [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+<../product_supplierinfo_for_customer>`_
```
Comment (elicoidal) [inline product_customer_code/README.rst]: I dont think the URL can work all the time. You need to specify absolute path for both.

**Review by elicoidal — CHANGES_REQUESTED** [review]
some details

File: product_customer_code/README.rst (hunk 2 above)
Comment (ysantiago) [reply product_customer_code/README.rst]: @elicoidal The suggested changes were applied.

**Review by elicoidal — APPROVED** [review]

**Pull request merged**